codecparsers/h265Parser.cpp

```diff
@@ -180,9 +209,9 @@ bool NalUnit::parseNaluHeader(const uint8_t* data, size_t size)
     // forbidden_zero_bit
     br.skip(1);
 
-    nal_unit_type = br.read(6);
-    nuh_layer_id = br.read(6);
-    nuh_temporal_id_plus1 = br.read(3);
+    READ_BITS(br, nal_unit_type, 6);
```

Comment:
To make code more clear, we do not need pass br to READ_BITS, you can refer to h264